src/lib/server/runtime-config.ts

```diff
@@ -85,7 +85,7 @@ const baseConfig = {
 	viewportFor: 'no-one' as 'employee' | 'no-one' | 'visitors' | 'everyone',
 	viewportContentWidth: 1000,
 	socialNetworkIcons: [] as Array<{ name: string; svg: string; href: string }>,
-
+	cartPreviewReadOnly: false,
```

Comment:
Rename the parameter, it should be 
```suggestion
	cartPreviewInteractive: false,
```